R/parallel.R

```diff
@@ -110,7 +107,7 @@ default_num_cpus <- function() {
   if (!is.null(ncpus)) {
     ncpus <- suppressWarnings(as.integer(ncpus))
     if (is.na(ncpus)) {
-      abort("`getOption(Ncpus)` must be an integer")
+      cli::cli_abort("{.fn getOption}({.arg Ncpus}) must be an integer.")
```

Comment:
This is weird, maybe use `{.code ... ` instead? Or at least double quote the option name?